client/src/components/NavBar.jsx

```diff
@@ -46,8 +46,8 @@ const mapStateToProps = (state) => {
   };
 };
 
-const mapDispatchToProps = (dispatch) => ({
-  doLogout: () => dispatch(doLogout()),
+const mapDispatchToProps = (dispatch, ownProps) => ({
```

Comment:
This made me realize, does our logout redirect to home?

Reply:
yes, the `onLogout` is a success callback that redirects to home. Unfortunately we need access to the react router context to do redirects. Since `NavBar` isn't directly beneath a route it doesn't get the router context, so I just passed a callback from `App.jsx` that does the redirect.